agent/functions/guard_test.go

```diff
@@ -0,0 +1,75 @@
+package functions
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestGuardPath(t *testing.T) {
```

Comment:
from Test Security Reviewer
パストラバーサルのテストケースをより強化するため、以下のようなimportを追加する必要があります。net/urlはURLエンコードされたパスのテスト用、unicode/normはUnicode正規化のテスト用です。

```suggestion
import (
	"strings"
	"testing"
	"net/url"
	"unicode/norm"
)
```